src/schema/notifications.ts

```diff
@@ -302,8 +322,12 @@ export class ZimbraNotifications {
 		if (items) {
 			items.forEach((i: any) => {
 				const item = normalizeFolder(i);
+				let itemId = item.id;
+				if (i.id.includes(':')) {
+					itemId = this.findSharedItemId(i);
```

Comment:
Can we just pass id to this function? Do we need any other attributes from item?

Reply:
Done.